server/sonar-server/src/main/java/org/sonar/server/es/NewIndex.java

```diff
@@ -317,10 +326,10 @@ private NestedFieldBuilder setProperty(String fieldName, Object value) {
       return this;
     }
 
-    public NestedFieldBuilder addStringField(String fieldName) {
+    public NestedFieldBuilder addKeywordField(String fieldName) {
       return setProperty(fieldName, ImmutableMap.of(
-        "type", "string",
-        "index", "not_analyzed"));
+        "type", FIELD_TYPE_KEYWORD,
+        "index", INDEX_SEARCHABLE_FOR_KEYWORD));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ATrailingCommentCheck)